docker/package/model.py

```diff
@@ -557,10 +557,12 @@ def __init__(
         target_networks: List[str],
         network_protos: Dict[str, List[str]],
         meta: PackagesMeta,
+        additional_native_deps: List[str],
```

Comment:
Since this is now no longer exclusive to `TezosBinaryPackage`, wouldn't be better to add this to `AbstractPackage` instead with an empty list as a default value?

Reply:
How it would help? We use that field in class constructor and still have to specify it in `__init__` consturtor

Reply:
that's a good point, I was thinking it could have saves some unnecessary repetition.

Never mind then :ok_hand: